site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -323,13 +331,16 @@
                 $(error_div_id).text("Please enter a valid new grader that is in the course and not already in the matrix.");
             } else if (self_grading) {
                 $(error_div_id).text("Please note that student is not able to grade themselves.");
+            } else if (team_grading) {
+                $(error_div_id).text("Please note that student is not able to grade their team members.");
```

Comment:
Isn't all javascript supposed to be in single quotes and not double quotes?

Reply:
Sorry for the delay.

and Yes, almost all the js files sticking with the single quotes
So, changed all of the double quotes to single.